src/routing/wrappers/Layout.svelte

```diff
@@ -13,6 +13,15 @@
   import routes from '../../config/routes';
 
   $: isProbeDetailsView = $store.route.section === 'probe';
+
+  const defaultTitle = 'GLAM: Glean Aggregated Metrics Explorer';
+  $: title =
+    $store.route.section === 'probe'
+      ? `${$store.probeName} | GLAM`
+      : defaultTitle;
+  $: {
+    document.title = title;
```

Comment:
So I'm not a svelte expert, but is this necessary? Shouldn't the `svelte:head` part set document.title?

Reply:
It should, yes. Let me test without that and see. I added that before I move the title changes to the script tag after learning that svelte has issues with updating the `<head>` inside an `{#if ...}` clause.

Reply:
Updated to remove that. It works just fine without it. Thanks!